Add note part for ODB usage (#7279)
MicrosoftDocs/microsoft-365-docs: microsoft-365/admin/activity-reports/onedrive-for-business-usage-ww.md

|Files  <br/> |The number of files in the OneDrive. <br/>|
|Active files  <br/> | The number of active files within the time period.<br/> NOTE: If files were removed during the specified time period for the report, the number of active files shown in the report may be larger than the current number of files in the OneDrive. >  Deleted users will continue to appear in reports for 180 days.  <br/> |
|Storage used (MB)  <br/> |The amount of storage the OneDrive uses in MB. |
|||

> [!NOTE]

> > The OneDrive for Business usage report is only available for OneDrive for Business that are associated with users who have licenses.

Reviewer: I would recommend update it to something like the report only includes users who have valid OneDrive for Business license.

> > The OneDrive for Business usage report is only available for OneDrive for Business that are associated with users who have licenses.

Reviewer 2: ```suggestion
> The OneDrive for Business usage report only includes users who have a valid OneDrive for Business license.
```
